tests/BlazingSQLTest/EndToEndTests/stringTests.py

```diff
@@ -196,6 +196,26 @@ def executionTest():
                 fileSchemaType,
             )
 
+            queryId = "TEST_10"
+            query = """SELECT
+                REPLACE(c_comment, 'in', '') as empty_replace,
+                REPLACE(c_comment, 'the', '&&') as and_replace,
```

Comment:
Please also add some test queries where you are applying a nested transformation and you have REPLACE as the internal transformation and the exeternal transformation 

For example doing something like this:
REPLACE(SUBSTRING(c_comment, 2, 10), 'e', '&&')
SUBSTRING(REPLACE(c_comment, 'e', '&&'), 2, 10)
REPLACE(c_comment,  'a', '1') LIKE '%the%'

Reply:
Resolved by https://github.com/BlazingDB/blazingsql/pull/1193/commits/952e91232ba46a12b24828ca8be425b37856191c